hermes/validator/question_generator.py

```
@@ -123,6 +125,37 @@ async def try_with_tools():
+                    },
+                )
+                question = response.get('messages', [])[-1].content
+                d = None
+                if token_usage_metrics is not None:
+                    d = token_usage_metrics.parse(
+                        cid_hash, phase=Phase.GENERATE_QUESTION, response=response, extra={"round_id": round_id}
+                    )
+                    token_usage_metrics.append(d)
+                return question, d, None
+
+            except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)